systemtest-scripts/systemtest-keys.sh

```diff
@@ -1,5 +1,6 @@
 #!/bin/bash -eu
 
+cd ..
```

Comment:
Is it possible to change the directory in the calling script? A bit confusing to see a script that starts with changing directory up one.